lib/portage/tests/process/test_spawn_fail_e2big.py

```diff
@@ -12,7 +13,9 @@
 @pytest.mark.skipif(platform.system() != "Linux", reason="not Linux")
 def test_spawnE2big(capsys, tmp_path):
     env = dict()
-    env["VERY_LARGE_ENV_VAR"] = "X" * 1024 * 256
+    # Kernel MAX_ARG_STRLEN is defined as 32 * PAGE_SIZE
+    max_arg_strlen_bytes = 1024 * 32 * resource.getpagesize()
```

Comment:
The documentation for `resource.getpagesize()`  says that it returns the result in bytes, so we probably should

```suggestion
    max_arg_strlen_bytes = 32 * resource.getpagesize()
```